### ISSUE TITLE
Échec par victoire - défi de Rage

### ISSUE THREAD

**Issue by Andizia** [JulienNigon/SpellcasterUniversity-Bug-Report#1195]
Bonjour,

Mon problème concerne le niveau de la tortue en défi de Rage, depuis la dernière mise à jour. Déjà, le jeu a planté trois fois en pleine partie, ce qui m'a sérieusement ralentie dans ma campagne.

J'avais enfin réussi à atteindre le dernier niveau du défi de Rage, j'ai collecté 4 des 5 cartes demandées par la Tortue, et je lui ai demandé la dernière (la bénédiction du phare).

J'ai laissé traîner un peu parce que j'hésitais à faire la fin standard, mais finalement je me suis décidée et j'ai demandé à la tortue de m'emmener loin d'ici. J'avais déjà eu la fin alternative et elle m'avait beaucoup fait rire, j'avais hâte de recommencer.

Et soudain, le drame.

Le bouton [Finir la partie] s'affiche, signe que j'ai enfin gagné ce défi qui m'a fait stresser. Je clique dessus... La cinématique d'échec se lance ! Contrariée, mais pensant à une sort de 10% de chance d'échec, je hausse les épaules et je relance le dernier niveau, ravie du bonus de réputation que j'y gagne.

Cette fois, je fais la victoire standard, sans toucher aux cartes de la tortue. Je suis docilement les étapes, brûlant deux d'entre elles pour lesquelles je remplissais les conditions à l'avance -comme c'est pratique, et j'arrive enfin au bout du niveau. Je clique sur le bouton [Finir la partie]... Et la cinématique d'échec revient !

CEPENDANT, je viens de voir que j'avais quand même débloqué les cartes correspondant à la victoire du défi.

Je l'aime beaucoup, cette campagne, mais j'aimerais bien la finir et recommencer à jouer normalement sans le stress de la réputation qui dégringole en permanence. Je pense que la dernière mise à jour a détraqué la détection de victoire ou quelque chose comme ça.

Aidez-moi à sauver mes élèves s'il vous plaît !

**Comment by gummynounours** [comment]
j'ai le même problème. une partie campagne en tres difficile et en defi "potion" impossible de finir la parti en gagnant : ce me met la video de la defaite quel que soit la victoire. et ca ne me debloque pas le succes victoire en difficile sur steam :( 

apres il est possible que ca vienne d'un probleme de la mise a jour, j'ai fait la moitié de la campagne avant la mise a jour

**Comment by Neelan22** [comment]
Bonjour,
nous sommes vraiment désolés  pour ce bug que nous venons de découvrir et corriger. Nous sommes conscient de la frustration que peut générer ce genre de bug, et nous en excusons. Si cela vous intéresse, vous pouvez nous fournir vos sauvegardes et nous tricherons un peu en les modifiant pour vous les rendre juste avant la fin du dernier tableau, avec les moyens pour finir la partie avec la fin de la tortue.
Les sauvegardes sont rangées ici : 
C:\Users\user\AppData\LocalLow\Sneaky Yak Studio\Spellcaster University\SteamID\saves

**Comment by Andizia** [comment]
Bonjour,
Après mûre réflexion, et comme de toute façon j'ai changé d'ordinateur, j'ai décidé que les élèves de cette école pouvaient bien brûler éternellement. Si le bug a été corrigé c'est l'essentiel, je n'aurai qu'à faire de nouvelles parties ! Je vous remercie de vous être penchés sur le problème.

Sur un autre sujet et pour ne pas avoir l'impression de vous avoir dérangés pour rien, dans le même scénario j'avais un problème de professeurs et élèves coincés dans la salle des trophées à marcher droit dans le mur, ainsi que dans la prison à faire des allers-retours à l'entrée de la pièce. Y a-t-il une solution à leur stupidité ?

**Comment by Neelan22** [comment]
On essaye de corriger ça, mais ce n'est pas facile à tracker étant donner qu'en rechargeant la partie, les élèves se débloquent (du coup les joueurs ne peuvent pas nous envoyer leur sauvegarde avec le bug). Néanmoins, avec cette sauvegarde nous pourrions enquêter sur la structure du château. Si tu nous envoies ta sauvegarde, accompagne la d'une capture d'écran pour qu'on puisse voir où se situe le pb.
Merci pour ton aide !

**Comment by Andizia** [comment]
Malheureusement la sauvegarde semble avoir été écrasée par défaut quand la partie a recommencé après la cinématique d'échec. N'ayant rien à envoyer, j'abandonne pour cette fois, mais si le problème se présente à nouveau j'essaierai de penser à vous l'envoyer immédiatement.
Ce qui ne me laisse plus qu'un seul choix... Retourner jouer ! En espérant que toutes les cartes que j'ai débloquées avec les défis ont été transférées de mon vieux PC au nouveau...

**Comment by Neelan22** [comment]
oui, nous avons mis en place le steam cloud avec la dernière mise à jour. Mais si tu les as perdu, envoie nous ton fichier Meta.xml (rangé avec les sauvegardes), on pourra te les rendre.

**Comment by Andizia** [comment]
Comme je m'y attendais, dès le premier niveau de ma nouvelle partie (avec les nouvelles cartes que j'aime beaucoup), un de mes élèves a décidé de faire des allers-retours entre les deux dortoirs. Je vous envoie ça avant que ça ne devienne la nouvelle mode dans l'école (j'ai dû changer le format de la sauvegarde parce que le .xml ne passait pas, j'espère que ça marchera quand même).
![Elève coincé](https://user-images.githubusercontent.com/61276889/106113556-e909a980-614e-11eb-8fea-d252ad218c64.png)
[save.txt](https://github.com/JulienNigon/SpellcasterUniversity-Bug-Report/files/5885953/save.txt)

**Comment by Neelan22** [comment]
merci pour la save. Comme on s'y attendait, l'élève n'est plus bloqué après le re-chargement de la partie. On poursuit les investigations
